### ISSUE TITLE
Game Resets Video Driver, then it crashes

### ISSUE THREAD

**Issue by BeastlyGhost** [bunfan/beat-banger#55]
I've been trying to simply open the game like normal, it opens like it should but it stays on the "BB" Logo screen, then it crashes the video driver for a second, and stops responding, the game itself doesn't even close after crashing, it just shows a White Screen and the game's icon disappears, the only way I could close it, is by using the task manager
![BeatBanger_zTaHKSfjmR](https://user-images.githubusercontent.com/45212377/123898340-2b6c2800-d933-11eb-8b66-84012cafa552.png)
![BeatBanger_Dxh5rXoL76](https://user-images.githubusercontent.com/45212377/123898345-2e671880-d933-11eb-891d-850b4e7e0893.png)

it can't be a OS problem since i've just installed Windows 10 in this machine, but I guess I can never be so sure

Reproduce:
- Download the 64 bit itch.io build
- Unzip it and open the game
- wait, Video Driver should reset

OS: Windows 10 Pro (64 bits)
RAM: 4GB
GPU: AMD ATI Radeon 4250
CPU: AMD Phenom(tm) II X2 565 (3.40 GHz)

Some Notes:
• 32 bit version also crashes the video driver, but somehow you can fullscreen on it if you are quick enough
• Sound Driver crashes with the video driver
• Running as Administrator, or in Compatibility Mode, gives the same effect as previously

EDIT:
     I have some updates, I've tried once again, but using a different version of Windows
 
    basically in Windows 10 (21H1), it crashes the video driver (and the sound driver)
    but on WIndows 11 (Build 222000 and Build 222000.65), not only it crashes both drivers, but it also stops the Night Light 
    service, which is kinda interesting because this didn't happen on Windows 10 at all

**Comment by potaito-rat** [comment]
you're not alone, downloaded the game (multiple times) and the same happened to me

**Comment by Dunuty** [comment]
accidentaly closed this, my bad

**Comment by Dunuty** [comment]
So I have some updates, I think I might have found out why this happens, well the game runs in Godot Engine, which, by itself, doesn't support OpenGL 3.3 or older (in my case at least), Godot Engine supports OpenGL 3.4 and later, so that might be a problem for those who wanna play the game and their GPUs doesn't support it, one thing that would actually help is if this game was completely open-source, so people could actually modify the code for the game to run in OpenGL 3.3 or even 3.2 for example, 

well, time to get a new GPU, i'm gonna leave this open since there might be people out there that have a GPU which supports OpenGL 3.4 and can't actually run the game

**Comment by theGrames** [comment]
Is this bug still occurring?

**Comment by Gui-iago** [comment]
> Is this bug still occurring?

it was not exactly a bug, my old computer (and GPU) didn't support OpenGL 4.6, which means I couldn't play the game because I just had an old GPU, the fix was either creating a Linux Virtual Machine, or installing Linux on a separate HDD/SSD, and playing the game from there, wine on linux could handle it just fine
